### PR TITLE
Bugfix MTE-1278 [v117] Fix testLoginsListFromBrowserTabMenu

### PR DESCRIPTION
**Pull request by clarmso**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/MTE-1278)

## :bulb: Description
`testLoginsListFromBrowserTabMenu` has been failing since July 11. After I saw this PR (https://github.com/mozilla-mobile/firefox-ios/pull/15551/files) I decided to come up with another solution. We can see if the issue is a legit issue or not.

I have run the changes on the full functional test suite on my computer.

## :pencil: Checklist
You have to check all boxes before merging
- [ ] Filled in the above information (tickets numbers and description of your work)
- [ ] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed I updated documentation / comments for complex code and public methods

### CODE REVIEW

File: Tests/XCUITests/FxScreenGraph.swift [hunk 1]
```diff
@@ -561,6 +561,7 @@ func createScreenGraph(for test: XCTestCase, with app: XCUIApplication) -> MMScr
         screenState.gesture(forAction: Action.ToggleNoImageMode) { userState in
             app.otherElements.tables.cells.switches[AccessibilityIdentifiers.Settings.BlockImages.title].tap()
         }
+        screenState.tap(app.buttons["Done"], to: HomePanelsScreen)
```
Comment (isabelrios) [inline Tests/XCUITests/FxScreenGraph.swift:564]: Have you checked that other tests are not affected by this change? sometimes modifying the screengraph could affect other tests

**Comment by isabelrios** [comment]
Can we use this PR to fix another test that is failing in the full functional? it is just removing two lines.
This is the test `testTapSignInShowsFxAFromRemoteTabPanel`, that needs L110 and L111 removed.

There are still two tests failing in the SaveLogins TestSuite, one is part of the Smoketest and one on the Full Functional due to the locked screen shown before the biometric screen info:

- testCreateLoginManually (it passes in re-try, it is not blocking but making longer builds)
- testDeleteLogin

We need to clarify that issue asap so that we can update or have green builds again.

File: Tests/XCUITests/FxScreenGraph.swift (hunk 1 above)
Comment (clarmso) [reply Tests/XCUITests/FxScreenGraph.swift:564]: Yes, I have run all full functional tests on this change. I wondered why this line is needed, too.

File: Tests/XCUITests/FxScreenGraph.swift [hunk 2]
```diff
@@ -945,6 +946,7 @@ func createScreenGraph(for test: XCTestCase, with app: XCUIApplication) -> MMScr
 
     map.addScreenState(LoginsSettings) { screenState in
         screenState.backAction = navigationControllerBackAction
+        screenState.tap(app.buttons["Settings"], to: SettingsScreen)
```
Comment (clarmso) [inline Tests/XCUITests/FxScreenGraph.swift:949]: I wondered why this line is needed in the screengraph as well.

Comment (isabelrios) [reply Tests/XCUITests/FxScreenGraph.swift:949]: You may need to tap now in the Settings button to go back to Settings and from there Screengraphs knows how to go to other screens

**Comment by isabelrios** [comment]
Closing this PR because that test failure was addressed in a different PR

**Pull request closed without merging**